system-contracts/hardhat.config.ts

```diff
@@ -34,8 +34,9 @@ export default {
     ethNetwork: "http://localhost:8545",
   },
   solidity: {
-    version: "0.8.20",
+    version: "0.8.24",
     settings: {
+      evmVersion: "cancun",
```

Comment:
just in case, what features do we need cancun for?

Reply:
`TLOAD`, `TSTORE`